README.md

```diff
@@ -23,14 +24,26 @@ It's significant achievement when taking into account:
 * reduced Total Cost of Ownership with only one code base to maintain
 * naturally synchronized development process across all the teams
 (backend, presentation logic, frontend, mobile)
-* the same features release for all the platforms
+* the same features released for all the platforms
 * and the time-to-market for new value is shorter
 
-Just to name a few.
-Unlike in other cross-platform development methodologies, here UI developers are given full
-control over look and feel of their applications. I also believe that such an approach,
-abstracting application presentation logic from specific platform, brings much better overall
-architecture which pays off in the future.
+Just to name a few. Usually cross-platform development tools fall into these categories:
+
+* abstraction over native UI components and IO operations (specialized API
+accessible from any possible programming language which is later either interpreted
+or compiled for specific platform)
+* HTML+CSS forced as UI on mobile 
```

Comment:
The use of HTML+CSS on all platforms, discarding the possibile advantages offered by native solutions on mobile devices.